rest-api-guidelines/guides/api-testing-ci-environment.md

```diff
@@ -13,13 +13,6 @@ The following must be available in the CI environment before testing:
     v7.5.0
    ```
 
-2. **Ruby** runtime MUST be available in the CI environment:
```

Comment:
We don't need ruby anymore ? it's not needed by dredd ?

Reply:
For swaggerhub we don't really need anything not needed for the current programming enviornment. For CLI usage only cURL is required or the libraries prepared for jenkins (in case of ci/cd). Dredd is a node app, so needs nodejs with npm

Reply:
@zdne I think you can safely remove the part which mentions Ruby or any CLI. On the other hand we should include information on how to access Swaggerhub programmatically - so need to mention Swaggerhub API and where it is (either the reference on our local instance or the one online, even though it has slightly different base path). WDYT? (ping @Amzani)

Reply:
Yes go for it! as we have questions on how to do it by our teams.

Reply:
Added mention and link to SwaggerHub API